### PR TITLE
Kadir-JS-Week3

### PR DESCRIPTION
**Pull request by laperla11**

### CODE REVIEW

File: week-3-practise/1-map-array.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+*/
+
+var pounds = [55.99, 104.99, 80.99, 3.99, 14.99];
+
+function finance(pound) {
+  return "£" + pound;
+} //Complete this statement
+var poundWithSign = pounds.map(finance);
+
+console.log(poundWithSign);
```
Comment (martynamaron) [inline week-3-practise/1-map-array.js]: Well done! 👏

File: week-3-practise/2-filter-array.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/*
+Mix and ranking each team using .filter() function.
+*/
+
+var championsOrder = [[0, 4], [1, 2], [2, 3], null, [1], [3, 1], [4, 0]];
+
+var premierLeagueTeams = championsOrder.filter(indexPair => {
+  return indexPair !== null && indexPair.length;
```
Comment (martynamaron) [inline week-3-practise/2-filter-array.js]: `indexPair.length` doesn't return a boolean, it returns a number, I think this needs a fix :)

File: week-3-practise/2-filter-array.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+];
+var ranking = [1, 2, 3, 4, 5];
+
+var mix = premierLeagueTeams.map(function(index) {
+  var rank = ranking[index[0]];
+  var team = champions[index[1]];
+  return [rank, team];
+});
+
+console.log(mix);
```
Comment (martynamaron) [inline week-3-practise/2-filter-array.js]: Well done! 👏

File: week-3-practise/3-forEach-array.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+/* 
+Using ---.forEach--- write a function that multiply a number by it self.
+*/
+
+var figure = [7, 14, 23];
+var amount = [];
+
+figure.forEach(function(number) {
+  amount.push(number * number);
```
Comment (martynamaron) [inline week-3-practise/3-forEach-array.js]: Nice! :)

**Comment by laperla11** [comment]
> Good work on week-3, no mistakes

Thanks.

File: week-3-practise/2-filter-array.js (hunk 2 above)
Comment (laperla11) [reply week-3-practise/2-filter-array.js]: You are right. I made the correction.

**Pull request closed without merging**